ERROR 2025-06-24 15:05:56,795 log 1938333 134359799559872 Internal Server Error: /
Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/core/views.py", line 87, in HomeView
    if request.user.is_authenticated():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'bool' object is not callable
ERROR 2025-06-24 15:05:56,796 basehttp 1938333 134359799559872 "GET / HTTP/1.1" 500 75426
INFO 2025-06-24 15:11:03,082 autoreload 1958228 134166285858624 Watching for file changes with StatReloader
INFO 2025-06-24 15:13:16,077 autoreload 1963795 134682533582656 Watching for file changes with StatReloader
ERROR 2025-06-24 15:13:21,194 log 1963795 134682424370880 Internal Server Error: /
Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/core/views.py", line 87, in HomeView
    if request.user.is_authenticated():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'bool' object is not callable
ERROR 2025-06-24 15:13:21,195 basehttp 1963795 134682424370880 "GET / HTTP/1.1" 500 76865
WARNING 2025-06-24 15:13:21,264 log 1963795 134682424370880 Not Found: /favicon.ico
WARNING 2025-06-24 15:13:21,265 basehttp 1963795 134682424370880 "GET /favicon.ico HTTP/1.1" 404 10643
INFO 2025-06-24 15:16:12,592 autoreload 1971326 140419410122560 Watching for file changes with StatReloader
INFO 2025-06-24 15:16:59,689 autoreload 1973351 127616466720576 Watching for file changes with StatReloader
ERROR 2025-06-24 15:17:03,664 log 1973351 127616358016704 Internal Server Error: /
Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/core/views.py", line 87, in HomeView
    if request.user.is_authenticated():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'bool' object is not callable
ERROR 2025-06-24 15:17:03,665 basehttp 1973351 127616358016704 "GET / HTTP/1.1" 500 76844
INFO 2025-06-24 15:18:01,974 autoreload 1973351 127616466720576 /home/eezis/code/closecall/core/views.py changed, reloading.
INFO 2025-06-24 15:18:02,310 autoreload 1976141 136195567617856 Watching for file changes with StatReloader
INFO 2025-06-24 15:18:27,945 autoreload 1976141 136195567617856 /home/eezis/code/closecall/incident/views.py changed, reloading.
INFO 2025-06-24 15:18:28,284 autoreload 1977242 128977555953472 Watching for file changes with StatReloader
ERROR 2025-06-24 15:18:31,945 log 1977242 128977445316288 Internal Server Error: /
Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1035, in find_library
    return parser.libraries[name]
           ~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'staticfiles'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/core/views.py", line 133, in HomeView
    return render(request, 'home.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1097, in load
    lib = find_library(parser, name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1037, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'staticfiles' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
crispy_forms_field
crispy_forms_filters
crispy_forms_tags
crispy_forms_utils
debugger_tags
highlighting
i18n
indent_text
l10n
log
markdown_filter
rest_framework
static
syntax_color
tz
widont
ERROR 2025-06-24 15:18:31,947 basehttp 1977242 128977445316288 "GET / HTTP/1.1" 500 219820
INFO 2025-06-24 15:18:31,947 basehttp 1977242 128977445316288 - Broken pipe from ('127.0.0.1', 56958)
ERROR 2025-06-24 15:18:36,137 log 1977242 128977445316288 Internal Server Error: /
Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1035, in find_library
    return parser.libraries[name]
           ~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'staticfiles'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/core/views.py", line 133, in HomeView
    return render(request, 'home.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1097, in load
    lib = find_library(parser, name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eezis/code/closecall/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 1037, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'staticfiles' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
crispy_forms_field
crispy_forms_filters
crispy_forms_tags
crispy_forms_utils
debugger_tags
highlighting
i18n
indent_text
l10n
log
markdown_filter
rest_framework
static
syntax_color
tz
widont
ERROR 2025-06-24 15:18:36,138 basehttp 1977242 128977445316288 "GET / HTTP/1.1" 500 219820
INFO 2025-06-24 15:18:58,899 basehttp 1977242 128977445316288 "GET / HTTP/1.1" 200 10654
INFO 2025-06-24 15:19:33,065 autoreload 1980177 135719552227136 Watching for file changes with StatReloader
INFO 2025-06-24 15:19:35,677 basehttp 1980177 135719442966208 "GET / HTTP/1.1" 200 10654
INFO 2025-06-24 15:19:35,714 basehttp 1980177 135719442966208 "GET /static/images/LogInWithStrava.png HTTP/1.1" 200 3635
INFO 2025-06-24 15:22:24,901 basehttp 1980177 135719442966208 "GET /support/ HTTP/1.1" 200 7266
INFO 2025-06-24 15:22:26,411 basehttp 1980177 135719442966208 "GET /articles/ HTTP/1.1" 200 8288
INFO 2025-06-24 16:55:56,065 basehttp 1980177 135719442966208 "GET /accounts/login/ HTTP/1.1" 200 5485
INFO 2025-06-24 16:55:56,118 basehttp 1980177 135719442966208 "GET /static/images/LogInWithStrava@2x.png HTTP/1.1" 200 7617
INFO 2025-06-24 16:56:16,044 basehttp 1980177 135719442966208 "GET / HTTP/1.1" 200 10654
INFO 2025-06-24 16:56:33,900 basehttp 1980177 135719442966208 "GET / HTTP/1.1" 200 10654
INFO 2025-06-24 17:50:39,640 autoreload 2374636 123972818376512 Watching for file changes with StatReloader
INFO 2025-06-24 17:51:17,114 autoreload 2376264 131680485472064 Watching for file changes with StatReloader
